plugins/modules/sda_host_port_onboarding_workflow_manager.py

```diff
@@ -2130,6 +2438,226 @@ def get_delete_port_channels_params(self, port_channel_details, network_device_i
+
+        self.log("create_vlans_and_ssids_mapped_to_vlans: {0}".format(create_vlans_and_ssids_mapped_to_vlans))
+        self.log("update_vlans_and_ssids_mapped_to_vlans: {0}".format(update_vlans_and_ssids_mapped_to_vlans))
+        self.log("no_update_vlans_and_ssids_mapped_to_vlans: {0}".format(no_update_vlans_and_ssids_mapped_to_vlans))
+
+        # Log the updated VLANs and SSIDs details.
+        self.log("Requested Details: {0}".format(updated_vlans_and_ssids))
+
+        return (
+            create_vlans_and_ssids_mapped_to_vlans,
+            update_vlans_and_ssids_mapped_to_vlans,
+            no_update_vlans_and_ssids_mapped_to_vlans,
+            updated_vlans_and_ssids
+        )
+
+    def get_create_update_remove_vlans_and_ssids_mapped_to_vlans_params(self, create_update_remove_vlans_and_ssids_mapped_to_vlans):
+
+        create_update_vlans_and_ssids_mapped_to_vlans_params = {
+            "fabric_id": self.have.get("fabric_id"),
+        }
+
+        if create_update_remove_vlans_and_ssids_mapped_to_vlans:
+            create_update_vlans_and_ssids_mapped_to_vlans_params.update(
+                {"payload": create_update_remove_vlans_and_ssids_mapped_to_vlans}
+            )
+        else:
+            create_update_vlans_and_ssids_mapped_to_vlans_params.update(
+                {
+                    "payload": [
+                        {
+                            "vlanName": "",
+                            "ssidDetails": []
+                        }
+                    ]
+                }
+            )
+
+        return create_update_vlans_and_ssids_mapped_to_vlans_params
+
+    def create_update_remove_vlans_and_ssids_mapped_to_vlans(self, create_update_remove_vlans_and_ssids_mapped_to_vlans_params):
+        self.log(
+            "Initiating Add/Update/Delete of VLANs and SSIDs mapped to VLANs with parameters: {0}".format(
+                create_update_remove_vlans_and_ssids_mapped_to_vlans_params
+            ),
+            "INFO"
+        )
+
+        return self.get_taskid_post_api_call(
+            "fabric_wireless",
+            "add_update_or_remove_ssid_mapping_to_a_vlan",
+            create_update_remove_vlans_and_ssids_mapped_to_vlans_params
+        )
+
+    def get_create_update_vlans_and_ssids_mapped_to_vlans_task_status(self, task_id):
+        task_name = "Create/Update VLANs and SSIDs Mapped to VLANs Task"
+        create_task_name = "Create VLANs and SSIDs Mapped to VLANs Task Succeeded for following VLAN(s) and SSID(s)"
+        update_task_name = "Update VLANs and SSIDs Mapped to VLANs Task Succeeded for following VLAN(s) and SSID(s)"
+        msg = {}
+
+        # Retrieve the parameters for create/update vlans and ssids mapped to vlans
+        create_vlans_and_ssids_mapped_to_vlans = self.have.get("create_vlans_and_ssids_mapped_to_vlans")
+        update_vlans_and_ssids_mapped_to_vlans = self.have.get("update_vlans_and_ssids_mapped_to_vlans")
+
```

Comment:
self.log("Retrieved 'create_vlans_and_ssids_mapped_to_vlans': {}".format(create_vlans_and_ssids_mapped_to_vlans), "DEBUG")
    self.log("Retrieved 'update_vlans_and_ssids_mapped_to_vlans': {}".format(update_vlans_and_ssids_mapped_to_vlans), "DEBUG")

Reply:
dont think this is necessary, it is from get_have itself.